modules/apps/feature-flag/feature-flag-web/src/main/resources/META-INF/resources/js/FeatureFlagToggle.tsx

```diff
@@ -47,7 +49,12 @@ const FeatureFlagToggle = ({
 			);
 
 			if (response.ok) {
+				const responseData = await response.json();
```

Comment:
Can we add a new line here.

Reply:
Also for naming this, it's best to do a search in portal for usages of `await response.json();` and then go with the most common pattern that matches your use case. In this case there is only one instance of `responseData` and many instances of `responseJSON` or just `data`. Usually we want to follow the more common pattern.

Reply:
Thanks Evan, I'll go with `data`.
Fixed